includes/oauth/class-google-login.php

```diff
@@ -74,7 +74,7 @@ public static function register_api_endpoints() {
 	 */
 	public static function api_check_if_oauth_configured() {
 		if ( ! Google_OAuth::is_oauth_configured() ) {
-			Logger::log( 'OAuth not configured.' );
+			self::handle_error( __( 'Google OAuth is not configured.', 'newspack' ) );
```

Comment:
Translation domain should be `newspack-plugin`

This needs to be updated for all translations on this file.